### PR TITLE
Add missing occ wnd commands

### PR DESCRIPTION
**Pull request by mmattel**
Add missing occ wnd commands

Backport **only** to 10.7

### CODE REVIEW

File: modules/admin_manual/pages/configuration/server/app_commands/wnd_commands.adoc [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+[source,console]
+----
+wnd
+ wnd:listen                 Listen to smb changes and store notifications for later processing
+ wnd:process-queue          Process the notifications stored by the wnd:listen command
+ wnd:set-service-account    Disable server-side encryption and decrypt all files
+----
+
+== Configure the Listener
+
```
Comment (micbar) [inline modules/admin_manual/pages/configuration/server/app_commands/wnd_commands.adoc:19]: I would add an example like this https://github.com/owncloud/windows_network_drive/blob/f9303603ea030e9510d6f2682145177d15d1aa37/doc/owncloud-wnd-listen-SERVER-SHARE.service

for a systemd service, which would be the "real world" use case.

Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/app_commands/wnd_commands.adoc:19]: You are right regarding more informatio is needed.
But instead rewriting all what is defined in the WND documentation, I am in favor making an explaining note and reference to the wnd:listen section in the wnd doc. This doc is currently in update process, see: #2739

Comment (mmattel) [reply modules/admin_manual/pages/configuration/server/app_commands/wnd_commands.adoc:22]: I have made improvements and addressed the topic above.

**Comment by mmattel** [comment]
@micbar Please review again.

File: modules/admin_manual/pages/configuration/server/app_commands/wnd_commands.adoc [hunk 2]
```diff
@@ -0,0 +1,115 @@
+= Windows Network Drive (WND)
+
+Marketplace URL: {oc-marketplace-url}/apps/windows_network_drive[External Storage: Windows Network Drives]
+
+Integrate Windows and Samba/CIFS shared network drives as external storages. For details please see the
+xref:enterprise/external-storage/windows-network-drive_configuration.adoc[Windows Network Drive (WND)] documentation.
+
+The `wnd` commands configures the WND app.
```
Comment (phil-davis) [inline modules/admin_manual/pages/configuration/server/app_commands/wnd_commands.adoc:8]:
```suggestion
The `wnd` commands configure the WND app.
```

**Review by phil-davis — APPROVED** [review]

**Pull request merged**